detection_rules/rule.py

```diff
@@ -952,7 +952,12 @@ def _get_packaged_integrations(self, package_manifest: dict) -> Optional[List[di
                 datasets.update(set(str(n) for n in node if isinstance(n, kql.ast.Value)))
 
         if not datasets:
-            return
+            # windows and endpoint integration do not have event.dataset fields in queries
+            for integration_tag in self.metadata.integration:
+                if integration_tag in ["windows", "endpoint", "apm"]:
+                    packaged_integrations.append({"package": integration_tag, "integration": integration_tag})
+                else:
+                    return
```

Comment:
The `datasets` variable originally generated from parsing the query of a rule, more specifically `event.dataset`. As a result, the windows, endpoint and apm integrations have a static value in the package storage repository therefore we can manually specify it.